app/src/sharedTest/java/org/oppia/android/app/resumelesson/ResumeLessonFragmentTest.kt

```diff
@@ -197,6 +201,36 @@ class ResumeLessonFragmentTest {
     }
   }
 
+  @Test
+  fun testResumeLessonFragment_lessonDescriptionIsInRTL_isDisplayedCorrectly() {
```

Comment:
```suggestion
  fun testResumeLessonFragment_lessonDescriptionIsInRtl_isDisplayedCorrectly() {
```

Ditto elsewhere in PR (including for 'LTR' which should be 'Ltr').

Reply:
Done.